spec/support/unit/change_value.rb

```diff
@@ -107,5 +111,9 @@ def change_value(value, value_changer)
     def dummy_value_for_column
       Shoulda::Matchers::Util.dummy_value_for(column_type)
     end
+
+    def date_column
```

Comment:
Since this returns a boolean, should this be:

``` suggestion
    def date_column?
```

Reply:
Yep! Interesting that my linting setup missed that. Ty for the suggestion!

Reply:
Made this change on 16df8f1a